packages/@aws-cdk/aws-efs/lib/efs-file-system.ts

```diff
@@ -122,7 +123,7 @@ export interface FileSystemProps {
   /**
    * Defines if the data at rest in the file system is encrypted or not.
    *
-   * @default false
+   * @default - if '@aws-cdk/aws-efs:defaultEncryptionAtRest' is enabled, the default is true. Otherwise, the default is false.
```

Comment:
This will confuse new users, which are not likely to know about `context-cdk.json` flags. Lets add some docs